core/lib/zksync_core/src/api_server/web3/pubsub.rs

```diff
@@ -388,3 +408,20 @@ impl EthSubscribe {
         notifier_tasks
     }
 }
+
+#[async_trait::async_trait]
+impl EthPubSubServer for EthSubscribe {
+    async fn subscribe(
+        &self,
+        pending: PendingSubscriptionSink,
+        sub_type: String,
+        filter: Option<PubSubFilter>,
+    ) -> SubscriptionResult {
+        let self_ = self.clone();
+
+        self.runtime_handle
```

Comment:
Would this logic work w/o `runtime_handle`? At least judging by a comment near the `runtime_handle` field, it seems `jsonrpc`-specific.

Reply:
removed